### ISSUE TITLE
Prepare Translations.

### ISSUE THREAD

**Issue by MetalES** [MetalES/ZeldaES-Solarus-Edition#71]
Prepare bitmaps, so they would be ready for translators asap.

By bitmaps I mean things like this, menu and Action icons translation.
Translation are the same as Ocarina of Time / Majora's Mask submenus.

![pause_submenus_name](https://cloud.githubusercontent.com/assets/4288633/25174235/ff400782-24f6-11e7-914c-441d748dbde7.png)
![pause_submenus_name](https://cloud.githubusercontent.com/assets/4288633/25174247/04911cd0-24f7-11e7-85fd-0c7a603a8a6a.png)
![pause_submenus_name](https://cloud.githubusercontent.com/assets/4288633/25174258/0ecf0dba-24f7-11e7-8a77-b5e193846f5a.png)

Remaining languages are
Spanish
Italian

Russian
Japanese
Chinese

German is done (Menu translations, texts and caption is litteral and approximative translation)
![sans titre 19](https://cloud.githubusercontent.com/assets/4288633/25174590/23efa096-24f8-11e7-8f6a-17f4f7acc41e.png)

**Comment by MetalES** [comment]
@diarandor, do you know a better way of translating "Quest Status", in OOT, it is "Estado de la Aventura", which is too long for a title 
Google Translate gave me "Estado de Búsqueda" ...

Also does this looks right in term of translation ? I do take as base Ocarina of Time 3D for translating bitmaps
![pause_submenus_name](https://cloud.githubusercontent.com/assets/4288633/25175552/78c306a0-24fb-11e7-8fc1-d485817fd18e.png)

"Masques" isn't translated, this is another menu, unused in this project

**Comment by Diarandor** [comment]
The words of the image are correctly translated. "Estado de la búsqueda" sounds quite weird to me, but I have seen it in some translated game that I cannot remember now. The problem is that there is not a perfect equivalent translation into Spanish of the word "quest" in the context of videogames; switching language in wikipedia it appears as "Misión" (mission), and "misiones secundarias" is a perfect translation for "side-quests". Both translations "Estado de la Aventura" and "Estado de la Misión" sound ok as translations of "Quest Status", and these are probably the best translations IMO.

If those expressions are too long, I suggest you to use "Logros", which means "Achievements", but I am not sure if that will be ok for your menu if what you display there are not achievements.

**Comment by MetalES** [comment]
Yeah, achievement is a synonym of quest status, it's only a menu to see the progression you've made and things you've collected throughout your journey so yes, that fit, thanks !

It's time to do italian, japanese, chinese and Russian, the rest will be handled by anyone who wants other languages. Yet, PAL5 (fr, en, es, de, it) would be the languages available first.
